Source/carthage/Printer.swift

```diff
@@ -0,0 +1,40 @@
+import Foundation
+
+public protocol Printer {
```

Comment:
`internal` should be okay if we use `@testable import carthage` in `VersionCommandSpec.swift`